modules/SDL.lua

```diff
@@ -145,6 +562,47 @@ function SDL:StopSDL()
   sleep(1)
 end
 
+function SDL.ForceStopSDL()
+  if config.remoteConnection.enabled then
+    ATF.remoteUtils.app:StopApp(config.SDL)
+  else
```

Comment:
@SNiukalov Can we use more short and simple command [slay](
http://www.qnx.com/developers/docs/6.5.0/index.jsp?topic=%2Fcom.qnx.doc.neutrino_utilities%2Fs%2Fslay.htm) ?